test/kubemark/pre-existing/README.md

```diff
@@ -0,0 +1,45 @@
+# Kubemark Pre-existing Provider Guide
+
+**Pre-existing Kubemark Master**
```

Comment:
I read the doc and I'm lost. I'm afraid I still don't understand what you are trying to achieve.

The kubemark setup that we are using is:
1. some kubernetes cluster
2. a separate VM where kubemark master is running
3. hollow-nodes that are running on cluster from 1)

What you are trying to reuse?
I initially though that just 1. But now I have a suspicion that it may be both 1 and 2?

Anyway - no matter what you are trying to reuse, please clearly describe it, because to be honest I don't know.